### PR TITLE
Update ci

### PR DESCRIPTION
**Pull request by Algunenano**
- Adapt for PG12 and postgis 3.0. Will fail to install if postgis_raster is not installed.
- Disabled/removed the MVT functions (currently unused and not worth maintaining for now).

### CODE REVIEW

File: src/pg/sql/40_observatory_utility.sql [hunk 1]
```diff
@@ -1,3 +1,16 @@
+-- In Postgis 3+, geomval is part of postgis_raster
+-- Trying to workaround it by creating the type ourselves leads to other issues:
+-- - If we create it under public, then if we try to create postgis_raster afterwards it will fail (ERROR:  type "geomval" already exists).
+-- - If we create it under cdb_observatory, then things work until we install postgis_raster. At that moment depending on how
+-- the search_path is set (per call, function, user...) we start getting random errors since it's mixing public.geomval and 
+-- cdb_observatory.geomval (function cdb_observatory.obs_getdata(geomval[], json) does not exist)
```
Comment (manmorjim) [inline src/pg/sql/40_observatory_utility.sql:6]: I'm thinking about if those random errors will happen also in the` dataservices-api` extension after installing the `postgis_raster`. 

`dataservices-api` creates the type `geomval` in the schemas `cdb_dataservices_server` and `cdb_dataservices_client` therefore if we install `postgis_raster` we would get mixing geomval types in different schemas.

**Review by manmorjim — APPROVED** [review]
LGTM. Just added a question related to dataservices.

File: src/pg/sql/40_observatory_utility.sql (hunk 1 above)
Comment (manmorjim) [reply src/pg/sql/40_observatory_utility.sql:6]: BTW great comments!

Comment (Algunenano) [reply src/pg/sql/40_observatory_utility.sql:6]: I tested it and couldn't reproduce the same issue (https://github.com/CartoDB/dataservices-api/runs/545847459), but now that I see the logs I don't thing the PG12 run was correct (it should have an error from the extra line I added to install raster, so I'll revisit it.

I think when using `dataservices-api` we enforce its schema to be used first, so if we create geomval we'll always use the one created and not the one from postgis_raster, which should handle the issue, but as I said, I'll recheck it.

**Pull request merged**